app/controllers/users_controller.rb

```diff
@@ -0,0 +1,37 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to new_users_path
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+    if @user.update(user_params)
+      redirect_to :root
```

Comment:
Redirect to `edit_user_path` instead

Reply:
Hi, So when we try to edit a user and it's updated successfully, the page should redirect to that specific user which is updated or it can be redirected to the page where all the users can be shown. I just tried the way you suggested but in that way, the page is going nowhere. So what's the point of editing a user again if it's updated successfully?
Thanks for your support!